marker_api/src/ast/item.rs

```diff
@@ -42,7 +42,8 @@ pub trait ItemData<'ast>: Debug {
     fn visibility(&self) -> &Visibility<'ast>;
 
     /// This function can return [`None`] if the item was generated and has no real name
-    fn name(&self) -> Option<String>;
+    /// This function can return `None` if the item was generated and has no real name
+    fn ident(&self) -> Option<&Ident<'ast>>;
```

Comment:
Previous doc wasn't removed o.o

Also, I would leave the link on `None`

Reply:
Oh yeah, I had a conflict while rebasing that part and must have missed that :)